browser/scripts/peopleStore.js

```diff
@@ -41,15 +40,18 @@ PeopleStore.prototype.initialize = function() {
 			case 'uiActiveGraphChanged':
 				that.people[uid].activeGraphUid = payload.activeGraphUid
 				that.emit('activeGraphChanged', that.people[uid])
+				that.people[uid].lastSeen = Date.now()
```

Comment:
could move these up above the switch so that no matter what is dispatched, lastSeen for that person is updated